packages/@uppy/companion/src/server/helpers/utils.js

```diff
@@ -183,7 +183,7 @@ module.exports.prepareStream = async (stream) => new Promise((resolve, reject) =
           return
         }
 
-        reject(new StreamHttpJsonError({ statusCode: err.response.statusCode, responseJson }))
+        reject(new HttpError({ statusCode: err.response.statusCode, responseJson }))
```

Comment:
Are these related?

Reply:
yes i renamed the error to a more generic name so it can be reused for non "stream" related http errors without the reader thinking "Wtf does StreamHttpJsonError mean?"